diff --git a/node_modules/react-callout/src/callout.jsx b/node_modules/react-callout/src/callout.jsx
index 02f750a..13a7b51 100644
--- a/node_modules/react-callout/src/callout.jsx
+++ b/node_modules/react-callout/src/callout.jsx
@@ -1,4 +1,5 @@
-import React, {Component, PropTypes} from 'react';
+import React, {Component} from 'react';
+import PropTypes from 'prop-types';
 
 export default class Callout extends Component {
     render() {
